lighthouse-core/test/audits/uses-rel-preload-test.js

```diff
@@ -196,6 +196,36 @@ describe('Performance: uses-rel-preload audit', () => {
     });
   });
 
+  it(`shouldn't suggest preload for protocol blob`, () => {
```

Comment:
does this test fail without the fix? seems like it wouldn't have wastedMs anyhow

Reply:
maybe let's just leave a TODO to make creating fake lantern graphs easier, it's inline with the other tests in the file for now